system/kube-monitoring/charts/prometheus-frontend/templates/_node.alerts.tpl

```diff
@@ -0,0 +1,65 @@
+### General node health ###
+
+ALERT NodeHighCpuUsage
+  IF avg by(instance)(irate(node_cpu{mode="idle"}[5m])) < 0.2
+  FOR 3m
+  LABELS {
+    service   = "node",
+    severity  = "warning",
+    context   = "availability",
+    dashboard = "kubernetes-node?var-server={{`{{$labels.instance}}`}}"
+  }
+  ANNOTATIONS {
+    summary = "High load on node",
+    description = "The node {{`{{ $labels.instance }}`}} has more than 80% CPU load.",
+  }
+
+ALERT NodeClockDrift
+  IF abs(ntp_drift_seconds) > 0.3
+  FOR 3m
+  LABELS {
+    service = "node",
+    severity = "critical",
+    context = "availability",
+    dashboard = "kubernetes-node?var-server={{`{{$labels.instance}}`}}"
+  }
+  ANNOTATIONS {
+    summary = "High NTP drift",
+    description = "The clock on node {{`{{ $labels.instance }}`}} is more than 300ms apart from its NTP server. This can cause service degradation in Swift.",
+  }
+
+### Network health ###
+
+ALERT NodeHighNumberOfOpenConnections
+  IF node_netstat_Tcp_CurrEstab > 20000
+  FOR 3m
+  LABELS {
+    service = "node",
+    severity = "warning",
+    context = "availability",
+    dashboard = "kubernetes-node?var-server={{`{{$labels.instance}}`}}"
+  }
+  ANNOTATIONS {
+    summary = "High number of open TCP connections",
+    description = "The node {{`{{ $labels.instance }}`}} has more than 20000 active TCP connections. The maximally possible amount is 32768 connections.",
+    {{ if .Values.ops_docu_url -}}
+    playbook = "{{.Values.ops_docu_url}}/docs/support/playbook/k8s_high_tcp_connections.html",
```

Comment:
Does this work? I think it has to be a label not an annotation according to https://github.com/sapcc/helm-charts/blob/master/global/prometheus-alertmanager/slack.tmpl#L25, or?